haveaseat-api/haveaseat-api/Repositories/DeskRepository.cs

```diff
@@ -0,0 +1,19 @@
+using haveaseat.DbContexts;
+using haveaseat.Repositories.Interfaces;
+using Microsoft.EntityFrameworkCore;
+
+namespace haveaseat.Repositories;
+
+public class DeskRepository : IDeskRepository
+{
+    private readonly DataContext _context;
+    public DeskRepository(DataContext context)
+    {
+        _context = context;
+    }
+    public async Task<List<Desk>> GetAllDesks()
+    {
+        var desks = await _context.Desks.ToListAsync();
```

Comment:
Staramy się nie używać varow w przypadku gdy prawa strona nie jest jasno zdefiniowana 
Fakt że w tym przypadku widać co zwraca Metoda ale przy większej logice dobrym nawykiem jest używanie varow tylko w przypadku gdy od razu widać co to za typ 

Warto poczytać o tym 

https://intellitect.com/blog/when-to-use-and-not-use-var-in-c/